docs/pvacseq/features.rst

```diff
@@ -97,9 +97,9 @@ As a last filtering step, pVACseq applies the top score filter to only keep the
 for each variant. As with all previous filters, this filter can also be run
 :ref:`standalone <filter_commands>`.
 
-**Scoring of candidate neoepitopes**
+**Ranking of candidate neoepitopes**
 
-Filtered neoepitopes are :ref:`scored and ranked <score>` based on the binding affinity,
+Filtered neoepitopes are :ref:`ranked <score>` based on the binding affinity,
```

Comment:
Should we update this reference (`<score>`) as well?  (Changing it breaks old URLs, I guess, but keeping it keeps the word `score` around in them instead.)